src/main/java/com/echobox/api/linkedin/jsonmapper/LinkedInJsonMapper.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.jsonmapper;
+
+import static java.util.Collections.unmodifiableList;
+
+import com.echobox.api.linkedin.logging.LinkedInLogger;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * LinkedIn implementation of a JSON-to-Java mapper.
+ * 
+ * @author Joanna
+ *
+ */
+public class LinkedInJsonMapper extends DefaultJsonMapper {
+
+  private static Logger LOGGER = LinkedInLogger.getLoggerInstance();
+
+  /**
+   * Creates a JSON mapper which will throw
+   * {@link com.echobox.api.linkedin.exception.LinkedInJsonMappingException} whenever an error
+   * occurs when mapping JSON data to Java objects.
+   */
+  public LinkedInJsonMapper() {
+    super();
+  }
+
+  /**
+   * Creates a JSON mapper which delegates to the provided {@code jsonMappingErrorHandler} for
+   * handling mapping errors.
+   * 
+   * @param jsonMappingErrorHandler
+   *          The JSON mapping error handler to use.
+   * @throws IllegalArgumentException
+   *           If {@code jsonMappingErrorHandler} is {@code null}.
+   * @since 1.6.2
+   */
+  public LinkedInJsonMapper(JsonMappingErrorHandler jsonMappingErrorHandler) {
+    super(jsonMappingErrorHandler);
+  }
+
+  /**
+   * @see com.echobox.api.linkedin.JsonMapper#toJavaList(java.lang.String, java.lang.Class)
```

Comment:
package is wrong, in fact I dont't like this kind of javadoc comment...what does it give?

Reply:
Copied directly over form restFb, can remove it because it's not really adding anything